lib/tasks/send_presentation_notifications.rake

```diff
@@ -0,0 +1,7 @@
+require 'rake'
+
+task send_presentation_notifications: :environment do
+  Event.requiring_notification.each do
+    EventNotificationJob.perform(event)
```

Comment:
You probably want perform_later here. When there is no processing background in place, rails will always perform jobs inline, even if you asked it to `perform_later`.

This way we dont need to remember to change this when we add a processing background